software/webapp/src/main/java/brooklyn/entity/webapp/jboss/Wildfly8ServerImpl.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package brooklyn.entity.webapp.jboss;
+
+import brooklyn.config.render.RendererHints;
+import brooklyn.enricher.Enrichers;
+import brooklyn.entity.Entity;
+import brooklyn.entity.basic.Attributes;
+import brooklyn.entity.webapp.JavaWebAppSoftwareProcessImpl;
+import brooklyn.event.feed.http.HttpFeed;
+import brooklyn.event.feed.http.HttpPollConfig;
+import brooklyn.event.feed.http.HttpValueFunctions;
+import brooklyn.location.access.BrooklynAccessUtils;
+import brooklyn.util.guava.Functionals;
+import com.google.common.base.Functions;
+import com.google.common.net.HostAndPort;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+public class Wildfly8ServerImpl extends JavaWebAppSoftwareProcessImpl implements Wildfly8Server {
+
+    public static final Logger log = LoggerFactory.getLogger(Wildfly8ServerImpl.class);
+
+    private volatile HttpFeed httpFeed;
+
+    public Wildfly8ServerImpl(){
+        super();
+    }
+
+    public Wildfly8ServerImpl(@SuppressWarnings("rawtypes") Map flags){
```

Comment:
Don't need these constructors (just the no-arg constructor, so can miss them out entirely).

We only have them in other (older) entities for backwards compatibility reasons, where folk called the constructors directly. Now everything should go through the EntitySpec mechanism for creating an entity.